<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Lobby_Window</class>
 <widget class="QMainWindow" name="Lobby_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-image: url(:/ress/resources/lobby.png)
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>620</y>
      <width>681</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="ready_up">
       <property name="text">
        <string>Ready up</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="start_early">
       <property name="text">
        <string>Start early</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>120</y>
      <width>381</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Current players</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Players ready to play!</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLCDNumber" name="lcdCurrentPlayers"/>
     </item>
     <item row="1" column="1">
      <widget class="QLCDNumber" name="lcdReadyPlayers"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLobby">
    <property name="title">
     <string>Lobby</string>
    </property>
   </widget>
   <addaction name="menuLobby"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
